src/youtube2zim/__main__.py

```diff
@@ -8,7 +7,7 @@
 
 def main():
     # allows running it from source using python youtube2zim
-    sys.path = [str(pathlib.Path(__file__).parent.parent.resolve())] + sys.path
+    sys.path = [str(pathlib.Path(__file__).parent.parent.resolve()), *sys.path]
```

Comment:
Do we still need this hack? I think we don't when running via hatch run or installing in editable mode

Reply:
I never understood under which condition this hack was required.
At least I can do what I want without it, so I'm perfectly fine if we get rid of this.

Reply:
I removed this hack and from my tests it is fine, I can run `python src/youtube2zim` without issue.